tests/browser/paths.js

```diff
@@ -0,0 +1,55 @@
+// @ts-check
+const {exec} = require('child_process');
+const path = require('path');
+
+/**
+ * @returns {Promise<string>}
+ */
+async function getChromePath() {
+    if (process.platform === 'darwin') {
+        return '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome';
+    }
+    if (process.platform === 'win32') {
+        return `${process.env.PROGRAMFILES}\\Google\\Chrome\\Application\\chrome.exe`;
```

Comment:
Not sure what version of google chrome you have installed, but mine are located in the `Program Files (x86)` and not `Program Files`

Reply:
If you execute `process.env.PROGRAMFILES` in Node.js (e.g. run `node -e "console.log(process.env.PROGRAMFILES)"` in console), what does it return?

Reply:
![image](https://user-images.githubusercontent.com/25481501/95756337-f422f180-0ca5-11eb-9971-060f66df7f1b.png)

Reply:
Maybe I've fixed it, I didn't check it on Windows.

Reply:
But...
```
Error: spawn C:\Program Files (x86)\Firefox Nightly\firefox.exe ENOENT       
    at Process.ChildProcess._handle.onexit (internal/child_process.js:267:19)
    at onErrorNT (internal/child_process.js:469:16)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
```
I think it's better to check the absolute path and if the file exist rather than the `(x86)` folder

Reply:
Yes, that seems to be working

Reply:
It's fixed